modules/tinymce/src/plugins/table/test/ts/browser/command/MergeCellCommandTest.ts

```diff
@@ -35,68 +31,153 @@ UnitTest.asynctest('browser.tinymce.plugins.table.command.MergeCellCommandTest',
   const clearEvents = () => modifiedEvents = [];
   const defaultEvent = { type: 'tablemodified', structure: true, style: false } as TableModifiedEvent;
 
-  const testCommand = (editor: Editor, command: string, tests: MergeCellTest[]) => {
-    Tools.each(tests, (test) => {
-      clearEvents();
-      editor.getBody().innerHTML = test.before;
-      editor.selection.select(editor.dom.select('td[data-mce-selected]')[0], true);
-      editor.selection.collapse(true);
-      editor.execCommand(command);
-      LegacyUnit.equal(cleanTableHtml(editor.getContent()), test.after, test.message);
-      LegacyUnit.equal(modifiedEvents, test.expectedEvents);
-    });
+  const testMerge = (editor: Editor, test: MergeCellTest) => {
+    clearEvents();
+    editor.getBody().innerHTML = test.before;
```

Comment:
```suggestion
    editor.setContent(test.before, { format: 'raw' });
```

We shouldn't be doing this, as it should be using the APIs and it's something I've been fixing while doing the conversion.